packages/babel-plugin/src/utils/state-manager.js

```diff
@@ -26,15 +26,18 @@ type ModuleResolution =
       type: 'commonJS',
       rootDir: string,
       themeFileExtension?: string,
+      aliases?: any,
```

Comment:
The type are obviously wrong. I'm reading the rest of the code and I can help with the types as needed.